configure.in

```diff
@@ -7263,6 +7260,10 @@ if test "$USE_WIN32" = 1 ; then
         fi
     fi
 
+    if test "$wxUSE_DIB" = "yes"; then
+        AC_DEFINE(wxUSE_WXDIB)
```

Comment:
I thought up how to change to using wxUSE_IMAGES_IN_RESOURCES without as many code changes as I thought. I will have wxUSE_IMAGES_IN_RESOURCES=1 under WindowsOS else it will be zero. And, I will define wxHAS_IMAGES_IN_RESOURCES when it is 1; therefore will NOT have to update all the samples.

@vadz Is this adding of wxUSE_IMAGES_IN_RESOURCES approved by you?

I plan to have setup.h define it to 1 when it is undefined to allow the client to set whether it is 1 or 0.

Tim S.